src/hotspot/cpu/arm/universalNativeInvoker_arm.cpp

```diff
@@ -29,3 +29,17 @@ address ProgrammableInvoker::generate_adapter(jobject jabi, jobject jlayout) {
   Unimplemented();
   return nullptr;
 }
+
+RuntimeStub* ProgrammableInvoker::make_native_invoker(BasicType* signature,
```

Comment:
Well, I guess in that case each platform would still need to define a factory method for the platform specific impl, so that platform agnostic code can reach it, and that would then need to be stubbed on unsupported platforms.

But, either way, I think it's better to stick with the conventions used in other VM code, which is to stub things out.